content/password.md

```diff
@@ -30,11 +32,11 @@ First, bring up the <u>systemd-boot</u> menu by holding down <kbd>SPACE</kbd> or
 
 Once the recovery operating system boots, close out of the installation window or choose **“try demo mode”** (be sure not to choose any install or repair options, as this could result in data loss).
 
-## Ubuntu
+### Ubuntu
 
 If you are using Ubuntu we can boot from a live disk and [here](/articles/live-disk) are instructions for creating the live disk. Once the live disk has been created refer to this [article](/articles/boot-menu) for booting from the live disk.
 
-## Mounting the Installed OS
+#### Mounting the Installed OS
 
 Press <kbd><font-awesome-icon :icon="['fab', 'pop-os']"></font-awesome-icon></kbd>/<kbd><font-awesome-icon :icon="['fab', 'ubuntu']"></font-awesome-icon></kbd>+<kbd>T</kbd> to open a terminal, and type in these commands:
```

Comment:
Most systems don't use either the Ubutu or Pop themed super keys, including every currently availiable System76 product baring the Launch Heavy. Beyond this, ``super + T `` doesn't yield a terminal under Ubuntu, where the short cut is ``ctrl + alt + T``.